internal/metrics/metrics.go

```diff
@@ -0,0 +1,94 @@
+// Copyright Envoy AI Gateway Authors
+// SPDX-License-Identifier: Apache-2.0
+// The full text of the Apache license is available in the LICENSE file at
+// the root of the repo.
+
+package metrics
+
+import (
+	"sync"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+// Metrics holds all prometheus metrics
```

Comment:
please make sure every comment has a period at the end (we explicitly ask anyone to follow the existing style and this one is one exmaple).
```suggestion
// Metrics holds all prometheus metrics.
```

Reply:
here and elsewhere

Reply:
updated

Reply:
hey, still MANY places following this style

Reply:
updated. 
let's aim a GHA in my next PR :D